<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CParameterPluginUI</class>
 <widget class="QWidget" name="CParameterPluginUI">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Plugin</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbEnableLocalInputMethod">
     <property name="text">
      <string>Enable local input method</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="cbEnableUserName">
     <property name="text">
      <string>Enable use the system user as the user</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Password</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cbSavePassword">
          <property name="text">
           <string>Enable save password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbEnableViewPassword">
          <property name="text">
           <string>Enable view password</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="gpEncryptKey">
        <property name="title">
         <string>Key used to encrypt saved password:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Encrypt key:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leEncryptKey">
             <property name="inputMask">
              <string/>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::EchoMode::Password</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbEncryptKey">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset theme="eye-on"/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Prompt type:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbPromptNo">
             <property name="text">
              <string>No</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="rbPromptFirst">
             <property name="text">
              <string>First</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbSystemCredential">
        <property name="text">
         <string>Store password with system credential manager</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Name</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cbShowPrefix">
        <property name="text">
         <string>Show protocol prefix in name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="cbShowIPPort">
        <property name="text">
         <string>Only show ip:port in name</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Viewer zoom:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbViewZoom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QGroupBox" name="gbCaptureAllKeyboard">
     <property name="title">
      <string>Capture all keyboard</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QGroupBox" name="gpDesktopShortcutsSctipt">
        <property name="title">
         <string>Desktop shortcuts script</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Disable script:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leDesktopShortcutsDisableScript"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbDesktopShortcutsDisable">
             <property name="text">
              <string>Browser</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Restore script:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="leDesktopShortcutsRestoreScript"/>
           </item>
           <item>
            <widget class="QPushButton" name="pbDesktopShortcutsRestore">
             <property name="text">
              <string>Browser</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cbPromptAdminPrivilege">
     <property name="text">
      <string>Prompt administrator privilege</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>leEncryptKey</tabstop>
  <tabstop>pbEncryptKey</tabstop>
  <tabstop>rbPromptNo</tabstop>
  <tabstop>rbPromptFirst</tabstop>
  <tabstop>cbSavePassword</tabstop>
  <tabstop>cbEnableViewPassword</tabstop>
  <tabstop>cbShowPrefix</tabstop>
  <tabstop>cbShowIPPort</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
